### ISSUE TITLE
Error Import Chart Of Accounts

### ISSUE THREAD

**Issue by kangbobi** [frappe/erpnext#36660]
### Information about bug

Message
Error: Value missing for Account: Parent Account

### Module

accounts

### Version

13,14

### Installation method

manual install

### Relevant log output / Stack trace / Full Error Message.

```shell
Traceback (most recent call last):
  File "apps/frappe/frappe/core/doctype/data_import/importer.py", line 126, in import_data
    doc = self.process_doc(doc)
  File "apps/frappe/frappe/core/doctype/data_import/importer.py", line 193, in process_doc
    return self.insert_record(doc)
  File "apps/frappe/frappe/core/doctype/data_import/importer.py", line 212, in insert_record
    new_doc.insert()
  File "apps/frappe/frappe/model/document.py", line 264, in insert
    self._validate()
  File "apps/frappe/frappe/model/document.py", line 540, in _validate
    self._validate_mandatory()
  File "apps/frappe/frappe/model/document.py", line 891, in _validate_mandatory
    raise frappe.MandatoryError(
frappe.exceptions.MandatoryError: [Account, 11 - Aktiva Lancar]: parent_account
```

**Comment by nabinhait** [comment]
fixed via #38321

**Comment by nabinhait** [comment]
This is a validation, not an error, you need to enter parent account
